clarity/next/tests/sbtc-db_test.ts

```diff
@@ -0,0 +1,26 @@
+
+import { Clarinet, Tx, Chain, Account, types } from 'https://deno.land/x/clarinet@v1.7.1/index.ts';
```

Comment:
Should we have deno/clarinet v1 tests if we’re going g to use the new clarinet-sdk?

Reply:
No, should be sdk only

Reply:
Then those tests should be removed, I guess.

Reply:
Copy, so remove all .ts files in the /tests directory?

Reply:
All `*_test.ts` files, from what I see. The `contract.test.ts` uses the new clarity-sdk so it's fine.

Reply:
Removed